types/providers.d.ts

```diff
@@ -29,13 +29,13 @@ export interface OAuthConfig<P extends Record<string, unknown> = Profile>
   scope: string
   params: { grant_type: string }
   accessTokenUrl: string
-  requestTokenUrl: string
+  requestTokenUrl?: string
```

Comment:
This one is true. Only OAuth 1.0 providers will need this

Reply:
Ok, I'm gonna keep this one 👍🏼